src/components/base-address/factories/address-factory.ts

```diff
@@ -187,21 +187,22 @@ export function useCountryRegions (code: string): boolean {
 }
 
 export function formatAddress (address: AddressIF): AddressIF {
-  address.postalCode = address.postalCode?.toUpperCase() || ''
-  if (address.country === 'CA') {
+  let formattedPostalCode = address.postalCode?.toUpperCase() || ''
+  if (address.addressCountry === 'CA') {
     address.postalCode = address.postalCode.replace('-', ' ')
     if (address.postalCode.length > 4 && address.postalCode[3] !== ' ') {
       address.postalCode = address.postalCode.slice(0, 3) + ' ' + address.postalCode.slice(3,)
     }
   }
-  return {
-    country: address.country?.trim(),
-    street: address.street?.trim(),
-    streetAdditional: address.streetAdditional?.trim(),
-    city: address.city?.trim(),
-    region: address.region?.trim(),
+  return{
+    addressCity: address.addressCity?.trim(),
+    addressCountry: address.addressCountry?.trim(),
+    addressRegion: address.region?.trim(), // FUTURE: change to undefined?
```

Comment:
Should this be `address.addressRegion?.trim()` ?

Why would you set this to undefined? (It will automatically be undefined if the optional chaining fails.)

Reply:
I believe the undefined is carry over from Assets Team, I will change these.

Reply:
Now it's `address.addressRegion?.trim()`:
https://github.com/bcgov/bcrs-shared-components/blob/4a37dd52699fa636fe23ba61beb0413179c74daf/src/components/base-address/factories/address-factory.ts#L200

Resolving.